crates/pink/runtime/src/runtime/pallet_pink.rs

```diff
@@ -167,6 +176,18 @@ pub mod pallet {
         pub fn set_treasury_account(account: &T::AccountId) {
             <TreasuryAccount<T>>::put(account);
         }
+
+        pub fn next_event_block_number() -> u64 {
```

Comment:
```suggestion
        pub fn take_next_event_block_number() -> u64 {
```

Indicate we are going to bump the next block number by one

Reply:
Done